manage.py

```diff
@@ -59,6 +59,7 @@ def init_hq_python_path():
         'ptop_preindex',
         'sync_prepare_couchdb_multi',
         'sync_couch_views',
+        'celery',
```

Comment:
i'm not sure i understand why this is needed - is this only applied when running compressor?